scriptmodules/libretrocores/lr-mame2003.sh

```diff
@@ -19,18 +19,19 @@ function depends_lr-mame2003() {
 
 function sources_lr-mame2003() {
     gitPullOrClone "$md_build" https://github.com/libretro/mame2003-libretro.git
-    # some games crash (eg pacmania) when built with -mfpu=neon-vfpv4 / -O3 on arm
-    isPlatform "arm" && sed -i "s/-O3/-O2/" Makefile
     # quieter build
     sed -i "s/-Wcast-align//" Makefile
 }
 
 function build_lr-mame2003() {
     make clean
+    isPlatform "rpi1" && params+=("ARM=1")
+    isPlatform "rpi2" && params+=("platform=rpi2")
+    isPlatform "rpi3" && params+=("platform=rpi3")
```

Comment:
We prefer not to use the cpu definitions in the makefiles - we handle those ourselves as it makes maintenance simpler.